cmd/geoipupdate/defaults_notwin.go

```diff
@@ -0,0 +1,12 @@
+// +build !windows
```

Comment:
I think `default_other.go` would be a more standard name for this, although I could be wrong.

Reply:
No idea, but that works!

Reply:
Also, not that it matters much, but I think the Windows one would generally have a suffix matching the build tag, i.e., `default_windows.go`

Reply:
https://github.com/golang/go/blob/50bd1c4d4eb4fac8ddeb5f063c099daccfb71b26/src/os/sticky_notbsd.go shows that you are in good company with `notwin` though :slightly_smiling_face: